drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -129,8 +129,10 @@ static void nrf5_get_eui64(uint8_t *mac)
 #if defined(CONFIG_TRUSTED_EXECUTION_NONSECURE) && defined(NRF_FICR_S)
 	soc_secure_read_deviceid(deviceid);
 #else
-	deviceid[0] = nrf_ficr_deviceid_get(NRF_FICR, 0);
-	deviceid[1] = nrf_ficr_deviceid_get(NRF_FICR, 1);
+	/* Use BLE.ADDR for setting EUI64, as this is currently the only feasible
```

Comment:
Please provide this implementation as an nRF54H20-only option, not replace the existing implementation that seemingly worked well for other devices so far.